website/docs/reference/resource-configs/grants.md

```diff
@@ -98,152 +70,45 @@ The `meta` config can also be defined under the `seeds` config block in `dbt_pro
 <File name='snapshots/schema.yml'>
 
 ```yml
-version: 2
-
 snapshots:
   - name: snapshot_name
-    config:
-      meta: {<dictionary>}
-
-    columns:
-      - name: column_name
-        meta: {<dictionary>}
-
+    config:  
+      grants:
+        select: ['reporter', 'bi']
 ```
 
 </File>
 
-The `meta` config can also be defined:
+The `grants` config can also be defined:
+
 - under the `snapshots` config block in `dbt_project.yml`
 - in a `config()` Jinja macro within a snapshot's SQL block
 
 See [configs and properties](configs-and-properties) for details.
 
 </TabItem>
-
-<TabItem value="analyses">
-
-The `meta` config is not currently supported for analyses.
-
-</TabItem>
-
-<TabItem value="macros">
-
-<File name='macros/schema.yml'>
-
-```yml
-version: 2
-
-macros:
-  - name: macro_name
-    meta: {<dictionary>}
-
-    arguments:
-      - name: argument_name
-        meta: {<dictionary>}
-
-```
-
-</File>
-
-</TabItem>
-
-<TabItem value="exposures">
-
-<File name='models/exposures.yml'>
-
-```yml
-version: 2
-
-exposures:
-  - name: exposure_name
-    meta: {<dictionary>}
-
-```
-
-</File>
-
-</TabItem>
-
 </Tabs>
 
 ## Definition
-The `meta` field can be used to set metadata for a resource. This metadata is compiled into the `manifest.json` file generated by dbt, and is viewable in the auto-generated documentation.
-
-Depending on the resource you're configuring, `meta` may be available within the `config` property, or as a top-level key. (For backwards compatibility, `meta` is always supported as a top-level key, though without the capabilities of config inheritance.)
-
-<Changelog>
 
-* `v0.16.0`: This property was introduced
-* `v0.21.0`: Introduced the `config` property, and gave `meta` the capabilities of a config
-
-</Changelog>
+You can use the `grants` field to set permissions or grants for a resource. These grants will be compiled into the `manifest.json` file complied by dbt, which you can view in the automatically generated documentation.
```

Comment:
> which you can view in the automatically generated documentation.

This isn't true (yet), so we shouldn't say it here. We could make it true with a little bit of HTML/CSS/JS in the `dbt-docs` site.

Everything up to this point, about compiling it into `manifest.json` / making it accessible as metadata, is true.

Reply:
Removed here: https://github.com/dbt-labs/docs.getdbt.com/pull/1642/commits/5ffaa49b6ecbc1525e0c3e8cb1c724e049bd5bdd

Is there a follow-up task to add this to the `dbt-docs` site?

Reply:
That can be food for future thought for the Core team—or maybe for the community member who opens the issue :)